specification/api/components/schemas/MIData.yml

```
@@ -0,0 +1,28 @@
+type: object
+properties:
+  data:
+    type: object
+    properties:
+      type:
+        type: string
+        const: ManagementInformation
+      attributes:
+        type: object
+        properties:
+          messageReference:
```

```suggestion
          specificationId:
```
I think we have concluded that MI should be (optionally) associated with a specification rather than individual messages/letters